### PR TITLE
Post-RC1 changes

### PR DESCRIPTION
**Pull request by nspark**
This PR includes the post-RC1 changes that I identified in my review. Some of these were noted in the RCM discussion, others were not.

### CODE REVIEW

File: content/shmem_pe_accessible.tex [hunk 1]
```diff
@@ -23,11 +23,11 @@
...
     specified \ac{PE} is accessible via \openshmem from the local \ac{PE}. The
-    \FUNC{shmem\_pe\_accessible} routine returns \CONST{TRUE} only if  the  remote
+    \FUNC{shmem\_pe\_accessible} routine returns a value indicating whether the remote
     \ac{PE} is a process  running from the same executable  file as the local
-    \ac{PE}, indicating that full \openshmem support for symmetric data objects
-    (that reside in the static memory and symmetric heap) is available, otherwise it
-    returns \CONST{FALSE}.  This routine may be particularly useful for hybrid
+    \ac{PE}, thereby indicating whether full support for symmetric data objects,
+    which may reside in either static memory or the symmetric heap, is available.
+    This routine may be particularly useful for hybrid
```
Comment (nspark) [inline content/shmem_pe_accessible.tex:30]: @jdinan I think this note about hybrid programming is better suited for the "Notes" subsection, but it also ties into whether the Altix references should be removed (see https://github.com/openshmem-org/specification/issues/161).

**Comment by nspark** [comment]
I tried to structure the commits in case you, as section chair, wanted some of them (e.g., macro stuff) but not others (e.g., text rewordings).

File: content/threads_intro.tex [hunk 2]
```diff
@@ -52,8 +52,8 @@
 In addition, a blocked thread will not prevent the progress of \openshmem calls
 performed on other \acp{PE}.
 
-\item In the \CONST{SHMEM\_THREAD\_MULTIPLE} thread level, all \openshmem calls are thread-safe,
-i.e., two concurrently running threads may make \openshmem calls and the outcome
+\item In the \CONST{SHMEM\_THREAD\_MULTIPLE} thread level, all \openshmem calls are thread-safe.
+Any two concurrently running threads may make \openshmem calls and the outcome
 will be as if the calls executed in some order, even if their execution is interleaved.
```
Comment (jdinan) [inline content/threads_intro.tex:57]: Does the "interleaved" semantic imply that if two threads in a PE perform blocking puts to the same buffer at a given target PE, that the result should be the data sent by one of the threads (i.e. not a mixture of the data from both or just garbage)?

File: main_spec.tex [hunk 3]
```diff
@@ -110,7 +110,7 @@ \subsubsection{\textbf{SHPALLOC}}\label{subsec:shpalloc}
 \subsubsection{\textbf{SHPCLMOVE}}\label{subsec:shpclmove}
 \input{content/shpclmove.tex}
 
-\subsubsection{\textbf{SHPDEALLOC}}\label{subsec:shpdealloc}
+\subsubsection{\textbf{SHPDEALLC}}\label{subsec:shpdealloc}
```
Comment (jdinan) [inline main_spec.tex:113]: Oh, wow, I didn't think it was possible for me to like the Fortran bindings less..

File: content/shmem_pe_accessible.tex (hunk 1 above)
Comment (jdinan) [reply content/shmem_pe_accessible.tex:30]: Thanks, I'll capture a note for this in https://github.com/openshmem-org/specification/issues/161.

**Comment by jdinan** [comment]
Thanks!

File: content/threads_intro.tex (hunk 2 above)
Comment (nspark) [reply content/threads_intro.tex:57]: No, that's still a data race and is not an artifact of threading. Even for a single threaded program, if a PE performs two put operations to the same destination buffer using two different source buffers without an ordering operation in between the puts, then the contents of the destination buffer are not well defined.

File: main_spec.tex (hunk 3 above)
Comment (nspark) [reply main_spec.tex:113]: When I first saw `SHPDEALLC`, I actually corrected the spelling on the API, only to realize that it was correct and the TOC entry was wrong. :sob:

**Pull request merged**